src/plugins/copyPaste/utils.js

```diff
@@ -20,7 +22,7 @@ export function arrayToTable(input) {
     }
 
     for (let column = 0; column < columnsLen; column += 1) {
-      tempElement.innerText = `${rowData[column] || ''}`;
+      tempElement.innerText = `${rowData[column] || (isNumeric(rowData[column]) ? rowData[column] : '')}`;
```

Comment:
I think that this changes should be extended to resolve the broader type of values (Booleans for example). 
Maybe this approach will be sufficient?
```suggestion
      tempElement.innerText = `${isEmpty(rowData[column]) ? '' : rowData[column]}`;
```

PS: Here I used `isEmpty` internal helpers.

Reply:
And if you agree with this solution please add tests for `true/false` values.